mmf/datasets/processors/bert_processors.py

```diff
@@ -206,3 +215,119 @@ def __call__(self, item):
             processed.segment_ids = processed.segment_ids.view(-1)
             processed.lm_label_ids = processed.lm_label_ids.view(-1)
         return processed.to_dict()
+
+
+@registry.register_processor("masked_multi_sentence_bert_tokenizer")
+class MaskedMultiSentenceBertTokenizer(BertTokenizer):
```

Comment:
Does this include the functionality provided by `MultiSentenceBertTokenizer`? If so can we remove the redundancy?